src/common/boundary_condition.cpp

```diff
@@ -123,4 +123,32 @@ void BoundaryCondition::apply(mfem::HypreParMatrix& k_mat_post_elim, mfem::Vecto
   eliminateToRHS(k_mat_post_elim, state.trueVec(), rhs);
 }
 
+const mfem::Coefficient& BoundaryCondition::scalarCoefficient() const
+{
+  SLIC_ERROR_IF(!std::holds_alternative<std::shared_ptr<mfem::Coefficient>>(coef_),
+                "Asing for a scalar coefficient on a BoundaryCondition that contains a vector coefficient.");
```

Comment:
s/Asing/Asking

Reply:
Now that all of our communication is virtual, I am realizing how many typos I make...